src/components/site/PageHeader.md

```diff
@@ -0,0 +1,34 @@
+### PageHeader
+
+```jsx
+<PageHeader
+  transparent="blue"
```

Comment:
based on the feedback above this could change to 

```jsx
className="blue"
```

Reply:
done